python/docs/source/getting_started/quickstart_connect.ipynb

```diff
@@ -28,7 +28,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "!$HOME/sbin/start-connect-server.sh --packages org.apache.spark:spark-connect_2.12:$SPARK_VERSION"
+    "!$HOME/sbin/start-connect-server.sh --packages org.apache.spark:spark-connect_2.13:$SPARK_VERSION"
```

Comment:
By the way, I made some changes because currently our scala version is `2.13`, `2.12` is no longer supported.